docs/jwst/extract_2d/main.rst

```diff
@@ -58,19 +58,19 @@ corresponding to the FITS keywords ``SLTNAME``, ``SLTSTRT1``, ``SLTSIZE1``,
 For NIRCAM GRISM and NIRISS WFSS :
 
 If the step parameter ``grism_objects`` is left unspecified, the default behavior
-is to use the source catalog that is specified in the input model's meta information, 
+is to use the source catalog that is specified in the input model's meta information,
 ``input_model.meta.source_catalog.filename``. Otherwise, a user can submit of list of
 ``GrismObjects`` that contains information about the objects that should be extracted.
 The ``GrismObject`` list can be created automatically by using the method in
 ``jwst.assign_wcs.utils.create_grism_bbox``. This method also uses the name of the source
 catalog saved in the input model's meta information. If it's better to construct a list
 of ``GrismObjects`` outside of these, the ``GrismObject`` itself can be imported from
-``jwst.transforms.models``. 
+``jwst.transforms.models``.
 
 
 Step Arguments
 ==============
-The extract_2d step has two optional arguments for NIRSPEC observations: 
+The extract_2d step has two optional arguments for NIRSPEC observations:
 
 * ``--which_subarray``: name (string value) of a specific slit region to
```

Comment:
Need to change this one too, from which_subarray to slit_name.

Reply:
done, thanks.